features/step_definitions/chef-repo.rb

```diff
@@ -4,12 +4,12 @@
     create_directory dir
   end
   # create a basic knife.rb
-  write_file "knife.rb", <<EOF
-local_mode true
-chef_repo_path '.'
-chef_zero.enabled true
-knife[:vault_mode] = 'client'
-EOF
+  write_file "knife.rb", <<~EOF
```

Comment:
Squiggly HEREDOC was added in Ruby 2.3 and therefore breaks in Ruby 2.2 which is currently part of the Travis test matrix.

How far back does chef-vault support?